### PR TITLE
Surround keys with quotes

### PR DESCRIPTION
**Pull request by Iwark**
Hi @tuvistavie !!

MySQL raises error when the keys contains reserved keyword like `order` such as:

```
INSERT INTO categories (id,order) VALUES (2, 2);
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order) VALUES (2, 2)' at line 1
```

Keys should be quoted to avoid this error:

```
INSERT INTO categories (`id`,`order`) VALUES (2, 2);
```

How do you think?

### CODE REVIEW

**Comment by danhper** [comment]
@Iwark Thanks for the fix, this is indeed needed!
The test failing seems to be related to Travis changing the way it builds Go dependencies, so I will fix this later.

Could you please add the `order` column to the `regions` table and modify the fixtures to reflect this so that we can check for this case in the tests?

Thank you! :smiley:

File: populator.go [hunk 1]
```diff
@@ -87,7 +87,8 @@ func (p *Populator) generateInsertStmt(fixture Fixture, data []map[string]interf
 		args = append(args, recordArgs...)
 	}
 	values := strings.Join(placeholders, ",")
-	return fmt.Sprintf(query, fixture.TableName, strings.Join(keys, ","), values), args
+	keysWithQuotes := surroundKeysWithQuotes(keys, p.Driver)
```
Comment (danhper) [inline populator.go]: You are modifying the slice in place here, so no need to use the return value,
and no need to return anything from the function either.

**Comment by danhper** [comment]
I just updated the build to test against all databases on the CI server. Could you please rebase and add the test for quoted keys to be sure it works with all the supported DBs.
Thanks!

**Comment by Iwark** [comment]
@tuvistavie 
I updated fixtures and added `order` column. Please review :)

**Comment by danhper** [comment]
Perfect, thank you very much! :smiley:

**Pull request merged**